.circleci/config.yml

```diff
@@ -634,7 +634,3 @@ workflows:
       - should-release-kmp-version:
           requires:
             - check-android-local-sdk-e2e-tests-result
-          filters:
-            branches:
-              only:
-                - main
```

Comment:
why was this removed?

Reply:
just for testing release pipelines in this branch. But the filter must be restored